### ISSUE TITLE
[GLPI10] ID not visible for CommonITILObjects in UI

### ISSUE THREAD

**Issue by VladoTTX** [glpi-project/glpi#11606]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.0

### Bug description

Previously in V9.5 it was possible to see the ID of CommonITILObjects (Tickets, Changes, Problems..) in the UI.
Example in V9.5.x:
![image](https://user-images.githubusercontent.com/71901245/168966679-193fe4ae-7c9f-4996-bf5e-7308ec2a7c62.png)

In GLPI V10 ID of the object is visible just in URL. Or you can enable IDs for all the objects in GLPI which will make ID part of name of all objects which is not always user need. Most important for most users is to know actual ID of ITIL Objects he is working on..

My suggestion would be to display it in title similar to this:
![image](https://user-images.githubusercontent.com/71901245/168966993-78e6f342-9080-495b-b5df-26741915b6e6.png)

It would make it always visible.
Example fix:
```
--- src/CommonGLPI.php.orig     2022-05-18 05:56:54.005628174 +0000
+++ src/CommonGLPI.php  2022-05-18 05:58:05.171919979 +0000
@@ -1044,7 +1044,7 @@
                   data-bs-toggle='tooltip' data-bs-placement='bottom'>
                   <i class='far fa-lg fa-list-alt'></i>
                </a>";
-
+           if ($this instanceof CommonITILObject)  echo $this->getTypeName(1) . " ID: " . $this->getID();
             echo "</div>";

             if ($this instanceof CommonITILObject) {
```

### Your GLPI setup information

10.0.0

**Comment by cconard96** [comment]
Related to #11493

**Comment by VladoTTX** [comment]
I would like just to highlight is related but not the same.
CommonITILObjects have object name visible in middle of Navigation header as shown on picture.
What users miss is the ID of Ticket, Problem, Change anywhere (and I would like to avoid displaying ID of all objects, by configuring the "show ID of GLPI objects" also in title everywhere)

Instead other (non-ITIL) objects, as noted in the #11493, have name visible just in the main tab. 
Yeah, it would be good to have it unified but I would vote also for having ID part this header for ITIL Objects.
Thanks.

**Comment by trasher** [comment]
Should be fixed by #11980

**Comment by VladoTTX** [comment]
Thanks, also this works fine for me.
